src/Commands/Module/ModuleGenerate.php

```diff
@@ -0,0 +1,222 @@
+<?php
+/**
+ * Copyright (c) Since 2020 Friends of Presta
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file docs/licenses/LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to infos@friendsofpresta.org so we can send you a copy immediately.
+ *
+ * @author    Friends of Presta <infos@friendsofpresta.org>
+ * @copyright since 2020 Friends of Presta
+ * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License ("AFL") v. 3.0
+ *
+ */
+
+namespace FOP\Console\Commands\Module;
+
+use FOP\Console\Command;
+use Nette\PhpGenerator\PhpFile;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Twig\Environment;
+
+class ModuleGenerate extends Command
+{
+    protected Filesystem $filesystem;
```

Comment:
Typed arguments are allowed only since php7.4 ( https://www.php.net/manual/en/language.types.declarations.php )
Could you please remove it ?

Reply:
fixed